File: backend/src/main/java/edu/cit/mabini/meditrack/dto/PatientDto.java
package edu.cit.mabini.meditrack.dto;

import lombok.*;

import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PatientDto {
    private Long id;
    private String patientNumber;
    private String firstName;
    private String lastName;
    private LocalDate birthDate;
    private String gender;
    private String address;
    private String contactNumber;
    private String emergencyContact;
    private Boolean archived;
}

File: mobile/app/src/main/java/edu/cit/mabini/meditrack/model/PatientDto.kt
package edu.cit.mabini.meditrack.model

data class PatientDto(
    val id: Long? = null,
    val patientNumber: String,
    val firstName: String,
    val lastName: String,
    val birthDate: String,
    val gender: String,
    val address: String,
    val contactNumber: String,
    val emergencyContact: String
)
